src/plugins/intel_gpu/src/graph/tile.cpp

```diff
@@ -22,7 +22,16 @@ layout tile_inst::calc_output_layout(tile_node const& node) {
 
     auto input_layout = node.input().get_output_layout();
     auto input_format = input_layout.format;
-    return layout{input_layout.data_type, input_format, desc->out_shape};
+
+    std::vector<int64_t> repeats = desc->repeats;
+
+    auto out_shape = input_layout.get_dims();
+    for (size_t i = 0; i < repeats.size(); ++i) {
+        out_shape[i] *= repeats[i];
+    }
+    out_shape = tensor(out_shape).sizes(input_format);
+
+    return layout{input_layout.data_type, input_format, tensor(out_shape)};
```

Comment:
It looks like `out_shape` is stored in bfyx-like format and tensor's c-tr will use this vector to fill `_sizes` sequentially which stores dimensions in `bfxyzw` format
So try to use `tensor(format fmt, const std::vector<value_type>& sizes, value_type default_size = 1)` instead